### PR TITLE
[WIP] Make VisualScript functions editable by double-clicking them

### PR DESCRIPTION
**Pull request by Jummit**
Fixes the oldest open VisualScript issue, #6072. Depends on the double click signal added by #28126.
There are some commits in this PR that I have to remove.

### CODE REVIEW

**Comment by fire** [comment]
@swarnimarun 's Google Summer of Code has this as a task. Want to work him on it?

**Comment by Jummit** [comment]
It's already done, I just have to sort out the git mess I made.

File: modules/visual_script/visual_script_editor.cpp [hunk 1]
```diff
@@ -42,6 +42,8 @@
 #include "visual_script_func_nodes.h"
 #include "visual_script_nodes.h"
 
+#include <iostream>
```
Comment (swarnimarun) [inline modules/visual_script/visual_script_editor.cpp]: Pretty sure this should not be here.

Comment (Jummit) [reply modules/visual_script/visual_script_editor.cpp]: Damn it, I always forget these.

Comment (swarnimarun) [reply modules/visual_script/visual_script_editor.cpp]: Weird to even have them there.
I use the Godot's print function or just a debugger instead(using a debugger is not always very easy though).

**Comment by Jummit** [comment]
Closing as #29681 will mess this up. Maybe I'll reopen it after it gets merged.

**Pull request closed without merging**